config/wrapStory.js

```diff
@@ -0,0 +1 @@
+export default function(story) { return story }
```

Comment:
you could just use `_.identity` instead of adding a new file

Reply:
Thats what I meant by this line:
> Alternatively, we can do something different inside preval to conditionally return an identity function if the wrapStory option is not set

Reply:
Yeah all I'm saying is I agree this seems overkill for now but I don't feel too strongly about it